python/GafferImageUI/CatalogueUI.py

```diff
@@ -152,6 +152,40 @@ def __statusIconColumnValueProvider( image, catalogue ) :
 registerColumn( "frame", "Frame", contextValueProvider( "frame" ) )
 registerColumn( "description", "Description", lambda image, _ : image["description"].getValue() )
 
+# Image properties
+
+def __resolutionColumnValueProvider( image, catalogue ) :
+	format_ = catalogue["out"].format()
+	value = "%d x %d" % ( format_.width(), format_.height() )
+	ratio = format_.getPixelAspect()
+	if ratio != 1.0 :
+		value += " [%d x %d]" % ( format_.width() * ratio, format_.height() )
+	return value
+
+def __channelsColumnValueProvider( image, catalogue ) :
+	return ", ".join( catalogue["out"].channelNames() )
+
+def __formatBox( box ) :
+	return "(%d %d) - (%d, %d)" % ( box.min().x, box.min().y, box.max().x, box.max().y )
+
+registerColumn( "image/resolution", "Resolution", __resolutionColumnValueProvider )
+registerColumn( "image/channels", "Channels", __channelsColumnValueProvider )
+
+registerColumn( "image/pixelAspectRatio", "Pixel AR",
```

Comment:
Maybe "Pixel AR" -> "Aspect"? Or "P. Aspect"? Isn't AR some new fangled thing the kids are using these days?

Reply:
Renamed to `P. Aspect` in https://github.com/GafferHQ/gaffer/pull/3688/commits/6c91dc3b6c9acfbb3dd02050157c9abb8f52992a#diff-f8b21d2701a672983fe238345811269bR218 to avoid confusion with the image's aspect ratio.